src/Storage/Storage.Management/help/Get-AzStorageBlob.md

```diff
@@ -201,6 +201,20 @@ testblob             BlockBlob 2097152         application/octet-stream       20
 This command gets a single blob with blob tag condition. 
 The cmdlet will only success when the blob contains a tag with name "tag1" and value "value1", else the cmdlet will fail with error code 412.
 
+### Example 10: Get blob properties
+```powershell
+(Get-AzStorageBlob -Container "ContainerName" -Blob "blob" -Context $ctx).BlobProperties.ImmutabilityPolicy
```

Comment:
How about first get the blob properties, then show the ImmutabilityPolicy.
This is useful when user want to get multiple properties of the single blob.
```suggestion
$blobProperties = (Get-AzStorageBlob -Container "ContainerName" -Blob "blob" -Context $ctx).BlobProperties
$blobProperties.ImmutabilityPolicy
```